generators/fastlane-setup/templates/fastlane/Fastfile

```diff
@@ -170,6 +176,21 @@ end
 #  ANDROID
 
 platform :android do
+  lane :set_keys do |options|
+    if ENV['ANDROID_APPCENTER_APP_SECRET'] then
+      appcenter_config = load_json(json_path: "android/app/src/main/assets/appcenter-config.json")
+      appcenter_config['app_secret'] = ENV['ANDROID_APPCENTER_APP_SECRET']
+      File.open("../android/app/src/main/assets/appcenter-config.json","w") do |f|
```

Comment:
We use a backup mechanism for this sort of stuff. In case of errors, it should revert to a previous state (see the javascript environment part)